doc/nrf/ug_matter_overview_dev_model.rst

```diff
@@ -28,3 +28,17 @@ The official Matter repository is fetched into the fork and the fork is included
 The fork is maintained and verified as a part of the nRF Connect SDK release process as an :ref:`OSS repository <dm_repo_types>`.
 
 For more information about Matter architecture and Matter in the |NCS|, read :ref:`ug_matter_overview_architecture_integration`.
+
+.. _ug_matter_overview_dev_model_ecosystems:
+
+Compatible ecosystems
+*********************
+
+One of the key features of the Matter protocol is the interoperability of different ecosystems it provides.
+Implementing support for Matter enables the :ref:`ug_matter_overview_multi_fabrics` and allows different vendor products to co-exist within the same Matter network.
+
+The following table lists market-proven solutions that have been tested with Matter in the |NCS| and are compatible with the nRF Connect platform and between themselves.
```

Comment:
I'm afraid to list the compatible ecosystems. The fact that we tested it with 3 commercial ecosystems does not mean that there are no 10 more which work as well. I would always keep the example ones, but then referred that our solution should work with any Matter 1.0 compatible ecosystem.